### PR TITLE
Merge pull request #8 from markielaz/ncnews-9

### PR DESCRIPTION
**Pull request by markielaz**

### CODE REVIEW

**Review by karlaferry — CHANGES_REQUESTED** [review]
Please add more error handling and make sure you check if article ID is valid and is existing. Then submit back for another review :)

**Review by karlaferry — CHANGES_REQUESTED** [review]
Almost there Mark! Just add one more test: test that you are getting an empty array of comments if the article exists but it does not have comments!

File: __tests__/app.test.js [hunk 1]
```diff
@@ -164,4 +164,55 @@ describe('GET /api/articles', () => {
...
+      expect(comments).toBeInstanceOf(Array);
+      expect(comments.length).toBe(11);
+      comments.forEach((comment) => {
+        expect(comment).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
```
Comment (Poonam-raj) [inline __tests__/app.test.js:186]: article_id is expected here and it should be the same for all comments (`1`)

File: app.js [hunk 2]
```diff
@@ -13,10 +13,36 @@ app.get("/api/articles", getArticles);
...
+  });
+
+  app.use((err, req, res, next) => {
+    if (err.code === '22003') {
+      res.status(404).send({ msg: 'Resource not found' });
+    } else next(err);
+  });
+  
+  app.use((err, req, res, next) => {
+    console.log(err);
```
Comment (Poonam-raj) [inline app.js]: get rid of console logs when you push up to github or make a PR

**Review by Poonam-raj — APPROVED** [review]
just add that article ID assertion into the test and you're all good to merge!
